src/loaders/express/helmet.ts

```diff
@@ -24,7 +25,7 @@ const helmetMiddlewares = () => {
     policy: 'strict-origin-when-cross-origin',
   })
 
-  const contentSecurityPolicyMiddleware = helmet.contentSecurityPolicy({
+  let cspDirectives = {
```

Comment:
fun fact: you can (and should) declare objects and arrays as `const` even if you are going to modify them later. this is because even though you modify the object, the _reference_ which JavaScript holds to the object remains the same, so `const` is okay as long as you don't reassign the variable to something else:
```
const a = {}
a = { somethingElse: 'somethingElse' } // Uncaught TypeError: Assignment to constant variable.
```

Reply:
thanks! good point - I edited accordingly